x-pack/legacy/plugins/apm/server/lib/transaction_groups/fetcher.ts

```diff
@@ -50,6 +51,37 @@ export function transactionGroupsFetcher(
     { '@timestamp': { order: 'desc' as const } }
   ];
 
+  const transactionsAggregation = {
+    transactions: {
+      terms: {
+        ...projection.body.aggs.transactions.terms,
+        order: { sum: 'desc' as const },
+        size: config['xpack.apm.ui.transactionGroupBucketSize']
+      },
+      aggs: {
+        sample: { top_hits: { size: 1, sort } },
+        avg: { avg: { field: TRANSACTION_DURATION } },
+        p95: { percentiles: { field: TRANSACTION_DURATION, percents: [95] } },
+        sum: { sum: { field: TRANSACTION_DURATION } }
+      }
+    }
+  };
+
+  const servicesAggregation = {
+    services: {
+      terms: {
+        field: SERVICE_NAME
```

Comment:
we should probably add something like size: 10000 here to make sure it queries all services. Otherwise it will be 10 by default.